CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ##### Bug Fixes
 
-* None.  
+* Fixed path checking for case-insensitive filesystems.
```

Comment:
trailing two spaces on this line, link to yourself, and then link to the CocoaPods issue (basically just following the format of the other changelog entries)